notebooks/MAST/dwarf_galaxy_search/dwarf_galaxy_search.ipynb

```diff
@@ -0,0 +1,832 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    
</pre><p>Split the cell here because the above this line is about target coordinates and the below is about building URL and API access</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>